core/controllers/topics_and_skills_dashboard_test.py

```diff
@@ -174,13 +174,105 @@ def test_post(self):
 
         json_response = self.post_json(
             feconf.SKILL_DASHBOARD_DATA_URL, {
-                'items_per_page': '10',
+                'num_items_to_fetch': '10',
                 'sort': 'Oldest Created'
             }, csrf_token=csrf_token)
 
         self.assertEqual(len(json_response['skill_summary_dicts']), 2)
         self.assertEqual(
             json_response['skill_summary_dicts'][0]['id'], self.linked_skill_id)
+        self.assertEqual(
+            json_response['skill_summary_dicts'][1]['id'],
+            self.subtopic_skill_id)
+        self.assertFalse(json_response['more'])
+        self.assertEqual(json_response['next_cursor'], None)
+
+        json_response = self.post_json(
+            feconf.SKILL_DASHBOARD_DATA_URL, {
+                'num_items_to_fetch': '10',
```

Comment:
Why is this a string? Shouldn't it be an int?

Ditto elsewhere.

Reply:
Fixed.